recipe/meta.yaml

```
@@ -10,35 +10,46 @@ source:
   sha256: 53686c3e912a33e8bf6975b878fd4bd0a7d589c8bad0a6a5573714af9e731dc2
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
+  skip: true  # [py<38]
 
 requirements:
   host:
-    - python >=3.7
+    - python
     - pip
     - setuptools_scm >=6.3.1
+    - wheel
```

```suggestion
    - wheel
    - setuptools
```

`setuptools` missing

---

https://github.com/iterative/telemetry-python/blob/0.0.9/pyproject.toml#L2

---

added